### PR TITLE
Use file builder from jaraco.path

### PR DESCRIPTION
**Pull request by jaraco**
## Summary of changes

The Setuptools test suite has a nice little tool for creating a small suite of
files (setuptools.tests.files.build_files). I'd like to use that functionalty
elsewhere, so I've extracted it into jaraco.path.

### Pull Request Checklist
- [X] Changes have tests

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -58,6 +58,7 @@ testing =
     pip>=19.1 # For proper file:// URLs support.
     jaraco.envs
     pytest-xdist
+    jaraco.path
```
Comment (benoit-pierre) [inline setup.cfg:61]:
```suggestion
    jaraco.path>=3.2.0
```

**Comment by jaraco** [comment]
The CI reveals that the dependency on `jaraco.path` adds a dependency on pyobjc on macOS, which fails to build on PyPy, so this change may not be viable.

**Comment by jaraco** [comment]
I think I fixed the issue with PyPy (by just excluding the dependency on PyPy), but now there's a new issue where `singledispatch` has deprecation warnings that are errors in Setuptools.

**Comment by jaraco** [comment]
Warnings with singledispatch fixed in the latest patch release, yet still PyPy failing. I must have gotten the environment marker wrong.

**Pull request merged**